### ISSUE TITLE
Switch front-end to next.js

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#1733]
Today I had to turn off the Netlify PR Preview builds for our next.js site.  This required me to switch our Vercel PR Preview builds from Gatsby to Next.

Because we're now basically using next everywhere, I think it's time to make the switch on staging/prod too.  For the 1.7 release, I'd like to fix any outstanding defects in the next frontend (things missing from Gastby), and then switch our deployment for production over to use src/frontend/next from src/frontend/gatsby.

On Tues at our triage, let's figure out all remaining issues that need fixing before Friday so we can make the switch.

**Comment by humphd** [comment]
Things I know about:

- Login button styling (#1703 is waiting on a rebase from @izhuravlev)
- Remove hero image from search and other non-home pages (I think @tonyvugithub was doing this in another PR?)
- Some colour differences in our theme (e.g., nav header is a different blue)
- Search page has a bunch of weird UI quirks with colours, sizes, and fonts
- Height of navbar is wrong

**Comment by tonyvugithub** [comment]
@humphd the hero image problem it will be fixed with #1674 PR

**Comment by HyperTHD** [comment]
Filed #1734 to deal with the navbar height issue

**Comment by izhuravlev** [comment]
Login button styling: #1703 is merged, but there is an issue with it displaying in Dark mode. PR #1777 by @tonyvugithub merged.
